ci/scripts/utils/ci_utils.sh

```diff
@@ -1,24 +1,104 @@
 #!/bin/env bash
 
-function cancel_slurm_jobs() {
-
-  # Usage: cancel_slurm_jobs <substring>
-  # Example: cancel_slurm_jobs "C48_ATM_3c4e7f74"
+function cancel_batch_jobs() {
```

Comment:
Scheduler determination should be independent of machine name.

We may want to move this elsewhere (maybe wxflow) later.
```suggestion
function determine_scheduler() {
  if command -v sbatch &> /dev/null; then
    echo "slurm";
  elif command -v qsub &> /dev/null; then
    echo "torque";
  else
    echo "unknown"
  fi
}

function cancel_batch_jobs() {
```

Reply:
Nice!